exporter/index.js

```diff
@@ -22,29 +22,34 @@ const validation = {
 };
 
 let sentryClient;
-const sentryIsEnabled = () => sentryClient == null;
+const sentryIsEnabled = () => !(sentryClient == null);
 
-if (process.env.SENTRY_DSN == null && process.env.SENTRY_RELEASE == null
-  && process.env.SENTRY_ENVIRONMENT == null && process.env.SENTRY_SERVER_NAME == null) {
-  console.log('Sentry not configured');
-} else {
+const enableSentry = !(
+  process.env.SENTRY_DSN == null
+  && process.env.SENTRY_RELEASE == null
+  && process.env.SENTRY_ENVIRONMENT == null
+  && process.env.SENTRY_SERVER_NAME == null);
+
+if (enableSentry) {
+  console.log('Init Sentry');
   Sentry.init({
     dsn: process.env.SENTRY_DSN,
     release: process.env.SENTRY_RELEASE,
     environment: process.env.SENTRY_ENVIRONMENT,
     serverName: process.env.SENTRY_SERVER_NAME,
   });
   sentryClient = Sentry.getCurrentHub().getClient();
-  console.log('Sentry initiated');
+} else {
+  console.log('Skipping Sentry');
 }
 
 const captureException = (error, runId = '') => {
-  console.error(`Exception captured for run ID: ${runId} -`, error);
-  if (sentryIsEnabled) Sentry.captureException(error);
+  console.error(`Exceptiqon captured for run ID: ${runId} -`, error);
```

Comment:
another split hair

Reply:
What's the issue?

Reply:
the spelling

Reply:
:-), yes that was an alternative spelling.